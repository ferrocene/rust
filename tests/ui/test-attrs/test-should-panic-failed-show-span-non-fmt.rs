//@ edition: ..2021
//@ compile-flags: --test
//@ run-flags: --test-threads=1 --nocapture
//@ run-fail
//@ check-run-results
//@ exec-env:RUST_BACKTRACE=0
//@ normalize-stdout: "finished in \d+\.\d+s" -> "finished in $$TIME"
//@ normalize-stdout: "TypeId\(0x[0-9a-f]+\)" -> "TypeId($$HEX)"
//@ needs-threads
//@ needs-unwind (panic)

#[test]
#[should_panic = "message"]
#[expect(non_fmt_panics)]
fn should_panic_with_substring_panics_with_non_string_value() {
    panic!(123);
}
