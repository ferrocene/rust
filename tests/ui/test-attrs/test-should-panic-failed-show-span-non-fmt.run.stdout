
running 1 test
test should_panic_with_substring_panics_with_non_string_value - should panic ... FAILED

failures:

---- should_panic_with_substring_panics_with_non_string_value stdout ----
note: expected panic with string value,
 found non-string value: `TypeId($HEX)`
     expected substring: "message"

failures:
    should_panic_with_substring_panics_with_non_string_value

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in $TIME

